eval/api_runner.py

```diff
@@ -32,6 +32,12 @@ def clean_generated_query(query: str):
     # if the string ` / NULLIF (` is present, replace it with `/ NULLIF ( 1.0 * `.
     # This is a fix for ensuring that the denominator is always a float in division operations.
     query = query.replace("/ NULLIF (", "/ NULLIF (1.0 * ")
+
+    # remove extra spaces around brackets especially for MySQL
+    query = query.replace(" ( ", "(").replace(" )", ")")
+    query = query.replace(" (", "(").replace(") ", ")")
+    query = query.replace("( ", "(").replace(" )", ")")
```

Comment:
Hello! Could we use regex here for cleaner code?

```python
result = re.sub(r'\s*\(\s*', '(', input_string)  # Remove spaces before and after '('
result = re.sub(r'\s*\)\s*', ')', result)        # Remove spaces before and after ')'
```

Reply:
Thanks, fixed- also realized we do not want to remove space after a closing brace. So, its this now:
```
# remove extra spaces around brackets especially for MySQL
query = re.sub(r"\s*\(\s*", "(", query)  # Remove spaces before and after '('
query = re.sub(r"\s*\)", ")", query)  # Remove spaces before ')'
```

Reply:
Cool, looks good to to me!

Reply:
looks good to me too! thanks for this change